### PR TITLE
ec2_instance - validate options on tower_callback

### PR DESCRIPTION
**Pull request by tremble**
Depends-On: https://github.com/ansible-collections/amazon.aws/pull/1202 

##### SUMMARY

- Validate options for tower_callback parameter
- Set `tower_callback.set_password` (the password) to `no_log=True`

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ec2_instance

### CODE REVIEW

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Review by alinabuzachis — APPROVED** [review]

File: plugins/module_utils/tower.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+import string
+import textwrap
+
+from ansible.module_utils._text import to_native
+from ansible.module_utils.six.moves.urllib import parse as urlparse
+
+
+def _windows_callback_script(passwd=None):
+    script_url = 'https://raw.githubusercontent.com/ansible/ansible/devel/examples/scripts/ConfigureRemotingForAnsible.ps1'
```
Comment (pkopylov) [inline plugins/module_utils/tower.py:15]: This url is not found. Is it OK?

**Comment by bastien-roucaries** [comment]
This is CVE-2022-3697

**Pull request merged**